app/assets/stylesheets/content.scss

```diff
@@ -1,8 +1,21 @@
 /************** CONTENT CONTAINER *******************************/
 
 #content {
+  background-color: #FFF;
+  margin-bottom: 6rem;
```

Comment:
Properties should be ordered background-color, font-family, margin-bottom, margin-top